platform_it/pom.xml

```diff
@@ -1057,6 +1058,7 @@
                                     -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=4025
                                     -Dsplice.olapServerOnStandAlone=true
                                     -enableassertions
+                                    -disableassertions:org.apache.datasketches.frequencies.ReversePurgeItemHashMap
```

Comment:
Do we still want this if we assume we shouldn't hit the assertion anymore?

Reply:
I had the same question. Since this assertion doesn't affect correctness, I thought it's safe to just disable it. If we encounter some performance issue, we could enable it locally and try if it gets hit. But I'm also fine with enabling it in general, no strong opinion here.

Reply:
The assertion is enabled in the latest commit.